packages/datadog-plugin-iovalkey/test/index.spec.js

```diff
@@ -0,0 +1,184 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { breakThen, unbreakThen } = require('../../dd-trace/test/plugins/helpers')
+const { ERROR_MESSAGE, ERROR_TYPE, ERROR_STACK } = require('../../dd-trace/src/constants')
+
+const { expectedSchema, rawExpectedSchema } = require('./naming')
+
+describe('Plugin', () => {
+  let Redis
+  let redis
+  let tracer
+
+  describe('iovalkey', () => {
+    withVersions('iovalkey', 'iovalkey', version => {
+      beforeEach(() => {
+        tracer = require('../../dd-trace')
+        Redis = require(`../../../versions/iovalkey@${version}`).get()
+        redis = new Redis({ connectionName: 'test' })
```

Comment:
```suggestion
  let Valkey
  let redis
  let tracer

  describe('iovalkey', () => {
    withVersions('iovalkey', 'iovalkey', version => {
      beforeEach(() => {
        tracer = require('../../dd-trace')
        Valkey = require(`../../../versions/iovalkey@${version}`).get()
        redis = new Valkey({ connectionName: 'test' })
```

Ideally the client instance is also renamed.